src/site/content/en/blog/origin-agent-cluster/index.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title: Requesting performance isolation with the Origin-Agent-Cluster header
+subhead: A new HTTP response header to limit domain-wide scripting and request dedicated resources from the browser.
+authors:
+  - domenic
+description: >
+  The Origin-Agent-Cluster header cuts off synchronous access to other origins on the same domain, and hints to the browser to give your origin dedicated resources.
+date: 2021-01-11
+hero: hero.jpg
```

Comment:
Please optimize this with tinyjpg.com (to keep the size of the repository down)

P.S. I like the "cluster" reference

Reply:
Thanks, and done! I feel like the guidance on https://web.dev/handbook/markup-media/ used to say no optimization, but now it's updated...